File: 2339.go
package main

import "fmt"

func main() {

	aluno := userInput()
	folhas := userInput()
	papel := userInput()

	if folhas < (aluno * papel) {
		fmt.Println("N")
	} else {
		fmt.Println("S")
	}

}

func userInput() int {
	var input int
	fmt.Scan(&input)
	return input
}

File: 2339.py

def main():
    inputs = input().split(" ")
    aluno = inputs[0]
    folhas = inputs[1]
    papel = inputs[2]

    if folhas < (aluno * papel):
        print("N")
    else:
        print("S")


if __name__ == "__main__":
    main()